test/cypress/support/getting_started.ts

```diff
@@ -0,0 +1,20 @@
+import { addCommands, GetCypressChainableFromCommands } from 'cloudify-ui-common/cypress/support';
+// import { waitUntilEmpty } from './resource_commons';
+
+declare global {
+    namespace Cypress {
+        // NOTE: necessary for extending the Cypress API
+        // eslint-disable-next-line @typescript-eslint/no-empty-interface
+        export interface Chainable extends GetCypressChainableFromCommands<typeof commands> {}
+    }
+}
+
+const switchGettingStarted = (username = 'admin', modalEnabled: boolean) =>
+    cy.cfyRequest(`/users/${username}`, 'POST', undefined, { show_getting_started: modalEnabled });
```

Comment:
Minor:

If we did

```suggestion
const switchGettingStarted = (modalEnabled: boolean) => (username = 'admin') =>
    cy.cfyRequest(`/users/${username}`, 'POST', undefined, { show_getting_started: modalEnabled });
```

Then we could declare the commands like so:

```ts
const commands = {
    enableGettingStarted: switchGettingStarted(true),
    disableGettingStarted: switchGettingStarted(false)
};
```

Reply:
Looks like `disableGettingStarted` was not used, so I ended up removing it